tests/test_classifier/__init__.py

```diff
@@ -0,0 +1,6 @@
+#Backend must be set before first use. 
+# Setting backend here allows us to run tests just in this folder, without running the whole yellowbrick.tests folder
+# This command will have no effect if backend has already been set previously. 
+import matplotlib
+matplotlib.use('Agg')
+
```

Comment:
When I tried to run only this folder of tests, it didn't let me because it didn't have proper backend. If you run the whole tests folder I think it is OK. If you include this line in the subfolder, it doesn't affect running the whole tests folder because it ignores later requests to set the back end. Let me know if you want me to remove, though.